src/reactions/aggregateReactions.js

```diff
@@ -43,4 +43,7 @@ export default (
         return reactionMap;
       }, new Map())
       .values(),
+  ).sort(
+    (r1: AggregatedReaction, r2: AggregatedReaction) =>
+      r1.users.length < r2.users.length ? 1 : -1,
```

Comment:
The comparer function does not need a `1` or `-1` but any positive or negative value will work.
Also, you need to pass `0` if equal.

The pattern to use is  to just return `r2.users.length - r1.users.length`

Reply:
Done 👍